PWGGA/GammaConv/AliAnalysisTaskOmegaToPiZeroGamma.cxx

```diff
@@ -2889,6 +2927,8 @@ void AliAnalysisTaskOmegaToPiZeroGamma::CalculateOmegaCandidates()
                 dropOutGammas_CALO.insert(secondGammaIndex);
               }
             }
+            delete pi0cand;
```

Comment:
This line and following ones: if a "new" operation is really not needed try to avoid it -> then we also don't have to worry about proper delete statements

Reply:
Tweaked the part added by me, to not need new and delete.